gen/calc.py

```diff
@@ -1105,6 +1108,8 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'mesos_container_log_sink': 'logrotate',
         'mesos_max_completed_tasks_per_framework': '',
         'mesos_recovery_timeout': '24hrs',
+        'mesos_seccomp_enabled': 'false',
+        'mesos_seccomp_profile_name': '',
```

Comment:
Since a default seccomp profile `default.json` is included, should we set the default value of this parameter to `default.json`?

Reply:
We could do that.
@Gilbert88 WDYT?

Reply:
No, we cannot do that. Do you forget the following case:

If a framework wants to enable seccomp but no default config. User could prefer a seccomp profile name if they need but by default all containers run without seccomp. How should the operator configure these two flags?

Reply:
@Gilbert88 In that case, I think the operator can set `mesos_seccomp_enabled` to true and set `mesos_seccomp_profile_name` to empty string, and framework can specify the specific seccomp profile name in ContainerInfo.

Anyway, I am OK with the current changes in this PR (i.e., set `mesos_seccomp_enabled` to false and set `mesos_seccomp_profile_name` to empty string), but I think we need to document these two parameters somewhere, otherwise how can operator know there is a default seccomp profile included in DC/OS installer package?

Reply:
@qianzhangxa , both ways sgtm. It is just inclusive VS exclusive.

@abudnik Qian's suggestion makes more sense, but we might have to add more logic to allow DC/OS to unset mesos_seccomp_profile_name as empty string `""`, and not passing the empty string to Mesos. Could you make that change?

Reply:
@qianzhangxa We'll document these flags and will give a short instruction on how to enable a default profile.
I'm leaning towards empty (disable) Seccomp profile by default - as it described in the design doc.

Reply:
If we step back, seems like in DC/OS, currently we don't support default config `unset` by passing empty string `""`.

Also, having the operator to explicitly set the profile name would allow the operator knows what exactly the default.json is.

Reply:
cc @qianzhangxa

Reply:
@Gilbert88 Yeah, that makes sense to me.